### PR TITLE
Limit property search results by 200.

### PR DESCRIPTION
**Pull request by pudiva**
Limits property search to 200 results and says when there is more.

https://trello.com/c/i3sRKruD/21-1st-line-of-address-search

### CODE REVIEW

File: app/apis/hackney_api/repairs_client.rb [hunk 1]
```diff
@@ -168,16 +168,17 @@ def get_property_by_postcode(postcode, min_level=LEVEL_NON_DWELL, max_level=LEVE
       )
     end
 
-    def get_property_by_address(address, min_level=LEVEL_NON_DWELL, max_level=LEVEL_ESTATE)
+    def get_property_by_address(address, min_level=LEVEL_NON_DWELL, max_level=LEVEL_ESTATE, limit: nil)
```
Comment (spannerinworks) [inline app/apis/hackney_api/repairs_client.rb]: is there a reason why we need a `nil` default? We could just make it mandatory. And get rid of the compact here and in the test stub.

**Review by spannerinworks — APPROVED** [review]
👍

File: app/apis/hackney_api/repairs_client.rb (hunk 1 above)
Comment (pudiva) [reply app/apis/hackney_api/repairs_client.rb]: well I thought that the `limit` parameter was kinda optional thing. If you pass `nil`, the API call won't have the parameter and then you'll get the default behaviour (which seems to be 100 results).

Comment (spannerinworks) [reply app/apis/hackney_api/repairs_client.rb]: I was thinking that since we never use the `nil` default we should just remove the code for it.

Comment (pudiva) [reply app/apis/hackney_api/repairs_client.rb]: done

Comment (pudiva) [reply app/apis/hackney_api/repairs_client.rb]: what do you think of this last patch?

File: app/models/hackney/property.rb [hunk 2]
```diff
@@ -29,8 +29,8 @@ def self.for_postcode(postcode)
     end
   end
 
-  def self.for_address(address)
-    HackneyAPI::RepairsClient.new.get_property_by_address(address)["results"].map do |attributes|
+  def self.for_address(address, limit: nil)
```
Comment (spannerinworks) [inline app/models/hackney/property.rb]: this one as well?

Comment (pudiva) [reply app/models/hackney/property.rb]: done

**Pull request merged**